app/assets.py

```diff
@@ -13,14 +13,12 @@
 )
 
 vendor_css = Bundle(
-    'vendor/semantic.min.css',
+    'vendor/*.css',
```

Comment:
The above comment I made related to page specific scripts and styles that we have functionality for in flask base already (it was some earlier PR I made). Also, I think it is just easier this way and much less of a pain in the future if we need to add site wide assets.